job-service-core/src/main/java/com/github/jobservice/core/queue/QueueServicesFactory.java

```diff
@@ -75,7 +76,17 @@ public static QueueServices create(final AppConfig configuration, final String t
     private static Connection createConnection(AppConfig configuration)
             throws IOException, TimeoutException, URISyntaxException, NoSuchAlgorithmException, KeyManagementException, AppConfigException
     {
-        return RabbitUtil.createRabbitConnection(configuration.getRabbitMQProtocol(), configuration.getRabbitMQHost(),
-                configuration.getRabbitMQPort(), configuration.getRabbitMQUsername(), configuration.getRabbitMQPassword());
+        final RabbitConfiguration rabbitConfiguration = new RabbitConfiguration();
```

Comment:
The alternative to directly using `RabbitConfiguration` would be to add another overloaded method of `RabbitUtil.createRabbitConnection` that takes the TLS protocol version, but that would require another release of the worker framework.

Since `util-rabbitmq` already has a dependency on `worker-configs`, I thought it was ok to do it this way, but if you prefer I can make the change to the Worker Framework instead?

Reply:
It is odd but as you say the dependency already existed.